app/packs/src/utilities/timezoneHelper.js

```diff
@@ -0,0 +1,34 @@
+const formatDate = (dateString) => {
+  const [datePart, timePart] = dateString.split(', ');
+  const [day, month, year] = datePart.split('.');
+  const [hours, minutes, seconds] = timePart.split(':');
+
+  // format date to js readable date
+  const formattedDate = `${year}-${month}-${day}T${hours}:${minutes}:${seconds}`;
+
+  const creationDateObj = new Date(formattedDate);
+
+  // local timezone offset
+  const timeZoneOffset = creationDateObj.getTimezoneOffset();
+  const offsetHours = Math.floor(Math.abs(timeZoneOffset / 60));
```

Comment:
to calculate the offset, simply use `Math.trunc` or `parseInt`. Those do the same but preserve the sign so you can get rid of the comparisons in line 19 and 23.

Reply:
corrected